kedro/extras/datasets/geopandas/__init__.py

```diff
@@ -1,4 +1,4 @@
-"""``GeoJSONDataSet`` is an ``AbstractVersionedDataSet`` to save and load GeoJSON files.
+"""``GeoJSONDataSet`` is an ``AbstractVersionedDataset`` to save and load GeoJSON files.
```

Comment:
```suggestion
"""``GeoJSONDataset`` is an ``AbstractVersionedDataset`` to save and load GeoJSON files.
```

Reply:
The non-core datasets aren't renamed in `kedro.extras` (and I think never will be, based on discussion with Merel previously, even though the same will be renamed in Kedro-Datasets, because these are already deprecated and slated for removal).